packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -591,7 +591,9 @@ export function makeWallet({
     const { brand } = await recP;
     if (!issuerToBoardId.has(issuer)) {
       const issuerBoardId = await E(board).getId(issuer);
-      issuerToBoardId.init(issuer, issuerBoardId);
+      if (!issuerToBoardId.has(issuer)) {
```

Comment:
Hmm, is this check needed twice?

Reply:
Yes.  There's an interleaving point before it.

Reply:
It's better done as a function, so I've extracted `initIssuerToBoardId`.